source/features/pagination-hotkey.tsx

```diff
@@ -21,19 +22,32 @@ const previousPageButtonSelectors = [
 function init(): void {
 	const createNextPageButton = select(nextPageButtonSelectors);
 	if (createNextPageButton) {
-		createNextPageButton.dataset.hotkey = 'ArrowRight';
+		createNextPageButton.dataset.hotkey = isRepoConversationList() ? 'ArrowRight' : 'n';
 	}
 
 	const createPreviousPageButton = select(previousPageButtonSelectors);
 	if (createPreviousPageButton) {
-		createPreviousPageButton.dataset.hotkey = 'ArrowLeft';
+		createPreviousPageButton.dataset.hotkey = isRepoConversationList() ? 'ArrowLeft' : 'p';
 	}
 }
 
 void features.add(__filebasename, {
-	shortcuts: {
-		'→': 'Go to the next page',
-		'←': 'Go to the previous page',
-	},
+	shortcuts: (isRepoConversationList())
+		? {
+			'←': 'Go to the previous page',
+			'→': 'Go to the next page',
+		}
+		: {
+			n: 'Go to the next page',
+			p: 'Go to the previous page',
+		},
+	include: [
+		// Activate only on pages with pagination
+		() => select.exists('.paginate-container'),
```

Comment:
I think we should just pick some pages in `github-url-detection` and specify them here

Reply:
I was thinking that the feature would automatically work on any pages that GitHub would add pagination in the future.

We do have good selector coverage ( which reminds me, there was another selector class choice for the notifications page that I wanted to suggest, but forgot ) for most cases, but then GitHub could beat us to it too though.